tests/handlers/tests.py

```diff
@@ -88,6 +91,17 @@ def test_invalid_multipart_boundary(self):
         self.assertEqual(response.status_code, 400)
 
 
+class TestASGIStaticFilesHandler(SimpleTestCase):
+    request_factory = RequestFactory()
+
+    @modify_settings(INSTALLED_APPS={'append': 'staticfiles_tests.apps.test'})
```

Comment:
Can you confirm what "that" is? If you mean doing `modify_settings`, without the decorator I end up 404'ing, presumably because it's not included in `INSTALLED_APPS`